recipes/ipython-memory-usage/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipython-memory-usage" %}
+{% set version = "1.1" %}
+{% set sha256 = "1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ianozsvald/ipython_memory_usage.git
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
This package might be a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages